### DISCUSSION TITLE
Install only the members area with my html5 website

### DISCUSSION THREAD

**Discussion by publiccoinboost** [e107inc/e107#4814] [General]
My website is finished and ready, html5 css3, but when I intaled e107 I lost it. How can I keep my website and only add the members area to it ? Is it even possible ? And if not, how should I add a members area to my html5 site ?

**Comment by Jimmi08** [comment]
Hi, 
you could rename your index.html and then set new frontpage in e107 admin area to your new index.html.  

or you can install e107 into subdirectory f.e. memberarea/
